### DISCUSSION TITLE
2.2.5-python3.8 for arm64

### DISCUSSION THREAD

**Discussion by mirisr** [apache/airflow#24767] [Q&A] [answered]
Hi, our development team is starting to use the M1 chip Mac machines, and we just upgraded airflow and our python. 

How do I go about getting an image for [2.2.5-python3.8](https://hub.docker.com/layers/airflow/apache/airflow/2.2.5-python3.8/images/sha256-d581907efc8ca601d72dcd41d6c1acef698e48abc2a96e4251ffcd7f50f1b098?context=explore) specifically for the platform linux/arm64.

I'm pretty new to docker, and I know there's this guide: https://airflow.apache.org/docs/docker-stack/build.html

But I really don't know what to do.

**Comment by potiuk** [comment, marked as answer]
Migrating to latest 2.3.3 for the whole company will be easier - and this is not a joke at all. 

See a long discussion here https://github.com/apache/airflow/issues/15635 - it took ~ year for us to make Airlfow 2.3.* finally working on ARM. The problem is that Airflow has ~ 600 dependencies and they all have to support ARM. Many of those dependencies only recently managed to add support and mostly only in newer versions - and we had to upgrade to a lot of those depencies in 2.3.0 in order to make it happen finally for 2.3.

Simply forget about making 2.2 ARM-compatible and focus on migrating your company to 2.3 if you need it.
